app/models/payment.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# Payments made by an agency to a provide for a particular site
+class Payment < UuidApplicationRecord
+  belongs_to :agency
+  belongs_to :site
+
+  validates :amount, numericality: { greater_than: 0.00 }
+  validates :care_finished_on, date_param: true
```

Comment:
For these fields can we also get presence validators? (amount, care_finished_on, care_started_on, paid_on)